src/serac/physics/utilities/functional/calculate_boundary_integration_points.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "serac/physics/utilities/functional/tensor.hpp"
+#include "serac/physics/utilities/functional/quadrature.hpp"
+#include "serac/physics/utilities/functional/finite_element.hpp"
+
+template <Geometry g, typename function_space, int Q, int dim>
```

Comment:
Not sure if this needs to be doxygened.

Reply:
oops, this file shouldn't be in here, this is part of a follow-on PR!